cypress/support.js

```diff
@@ -0,0 +1,4 @@
+import 'bootstrap/scss/bootstrap.scss'
```

Comment:
Make sure you add both `bootstrap` & `bootstrap-vue`. `bootstrap-vue` only contains vue specific styles.